Add first profile page (#1)
orcwg/.github: .github/profile/README.md

>  <img src="https://github.com/user-attachments/assets/e3e706de-2225-4242-84cd-a9dbe517c447" />

Reviewer: Please always make sure to include the file in the repo itself, like that we do not rely on external source for the content to be complete.

Reviewer 2: Done

> **Welcome to the [ORC](https://orcwg.org/) GitHub space** – an [Eclipse Foundation Working Group](https://eclipse.org/) working on supporting Open Source Community to be compliant with the demands of the *[Cyber Resilient Act](https://digital-strategy.ec.europa.eu/en/policies/cyber-resilience-act).* 

Reviewer: A few comments:

- I think Github calls this an _organization_ and that _(code)spaces_ are something else.
- wouldn't add links to external resources so high up in the page, as you risk having people navigate away instead of following the links on how to contribute.

```suggestion
**Welcome to the ORC GitHub organization** – ORC is an Eclipse Foundation Working Group whose immediate focus is to help open source maintainers and users meet the compliance requirements of the **European Cyber Resilience Act (CRA).**
```

Maybe we can add the links back down somewhere?

## The Cyber Resilience Act

The ORC WG main focus today is on supporting the open source community in developing assets that helps implement the CRA requirements. This has been translated into specific work streams as follows:

> - **[CRA FAQs](https://github.com/orcwg/cra-hub)**- Either you are newbie to the CRA, or a pro already working on its implementation, the CRA FAQ is a space for the open source community to ask questions and clarify key CRA topics..

Reviewer: ```suggestion
- **[CRA FAQs](https://github.com/orcwg/cra-hub)**- Either you are newbie to the CRA, or a pro already working on its implementation, the CRA FAQ is a space for the open source community to ask questions and clarify key CRA topics.
```

> - **[ORC Delivarable Plan](https://github.com/orcwg/orcwg/tree/main/cyber-resilience-sig#deliverables)** - The ORC community has outlined key deliverable as the key assets to work on.

Reviewer: ```suggestion
- **[ORC Delivarable Plan](https://github.com/orcwg/orcwg/tree/main/cyber-resilience-sig#deliverables)** - The ORC community has outlined key deliverables to work on in 2025.
```

> - **[Inventory](https://github.com/orcwg/cra-hub/blob/main/inventory.md)** - Inventory of resources relevant to the horizontal CRA standards, Open Source Steward obligations, and voluntary attestation program

Reviewer: ```suggestion
- **[Inventory](https://github.com/orcwg/cra-hub/blob/main/inventory.md)** - Inventory of resources relevant to the horizontal CRA standards, Open Source Steward obligations, and the security attestation program.
```

- **Join our calls**. Our [meetings](https://github.com/orcwg/orcwg/blob/main/MEETINGS.md) are public and open to all. ([iCal format](https://calendar.google.com/calendar/ical/c_7db8e3f13c4fac984103918a97c704bb1d619da0fdb66d33f1747849b6020aea%40group.calendar.google.com/public/basic.ics))
- **Subscribe to our [mailing List](https://accounts.eclipse.org/mailing-list/open-regulatory-compliance)** ([Archive](https://www.eclipse.org/lists/open-regulatory-compliance/maillist.html))
- **Join our [Slack](https://orcwg.slack.com/) channel** ([Request invitation](https://join.slack.com/t/orcwg/shared_invite/zt-2vi7gi5ad-re2b35i95ar3WaVF2zoZaA))

Some key ongoing activities in which you can engage with us:
### I'm a developer/maintainer in an open source project

- Here you have a few pointers that can help you in your work towards compliance:
	- [CRA FAQs for maintainers](https://github.com/orcwg/cra-hub/blob/main/faq.md#maintainers)
	- [Vulnerability handling specification](https://github.com/orcwg/vulnerability-management-spec) - we'd like to have you contributing to this specification. **This is the first obligation under the CRA**

### I work for an OSPO
- Specific actions for 

> 	- [CRA FAQs for manufacturers](https://github.com/orcwg/cra-hub/blob/main/faq.md#manufacturers)

Reviewer: I think we should push folks not only to the relevant parts of the FAQ but also to the relevant issues whether in the tracker, e.g.: https://github.com/orcwg/cra-hub/labels/Maintainer or the project board, e.g.: https://github.com/orgs/orcwg/projects/7/views/2

Reviewer 2: Added, but feel free to propose an alternative way of presenting it.

> 	- Attestation programs, SBOMs, due diligence for manufacturers ... there are multiple [deliverables](https://github.com/orcwg/orcwg/tree/main/cyber-resilience-sig#deliverables) waiting for your contributions.

Reviewer: ```suggestion
	- Attestation programs, SBOMs, due diligence for manufacturers… there are multiple [deliverables](https://github.com/orcwg/orcwg/tree/main/cyber-resilience-sig#deliverables) waiting for your contributions.
```

### I am a steward
- [CRA FAQs for Stewards](https://github.com/orcwg/cra-hub/blob/main/faq.md#open-source-software-stewards)
- [Vulnerability handling specification](https://github.com/orcwg/vulnerability-management-spec) - Contribute and stay up to date. **This is the first obligation under the CRA**

> ## Statistics (last 12 months)

Reviewer: I'm on the fence about these stats for now and I'm particularly hostile to those that promote vanity metrics like GH stars.

Reviewer 2: I agree, today that could be a distraction. I think we can get rid of them and focus on providing pointers to the key activities.

> # Welcome to Eclipse Open Regulatory Compliance Working  <img src="https://avatars.githubusercontent.com/u/189788385?s=200&v=4" width="200px"/>

Reviewer: Any chance we can get a nicer banner, here, @shanda-eclipse?

Reviewer 3: @tobie do you just need a background image for text to overlay? We have this but I can also make something else if you provide some further direction/specs.
![IMG_3105](https://github.com/user-attachments/assets/44a4b6b5-78f2-42cc-bc77-406623fcd2c2)

Reviewer: This is a markdown document. You can't overlay stuff.

Reviewer 3: @jrico-eclipse here is the banner we discussed 
![ORC Banner - general (2)](https://github.com/user-attachments/assets/e3e706de-2225-4242-84cd-a9dbe517c447)

* Compile guidelines and best practices for integrating compliance considerations into all stages of the software development lifecycle, from planning and design to deployment and maintenance.
* Collaborate with legal experts, industry stakeholders, regulatory bodies, and government officials to ensure alignment with current and evolving regulatory frameworks.
* Provide a forum where feedback can be garnered from government with respect to the specifications being developed, and that all interested stakeholders can meet when and how appropriate to better ensure the specifications meet the objectives established by government.
* Foster transparency and accountability within the open source community by promoting adherence to regulatory standards.
* Identify gaps in educational resources and training materials to support open source developers in understanding and implementing compliant development practices.
* With staff support, monitor regulatory updates and emerging trends to continuously refine and update the established processes and guidelines.
* Promote the adoption of the resulting specifications to various relevant standardisation bodies.

> * Through its member organisations, promote the adoption of compliant software development practices across industry through advocacy, outreach, and knowledge-sharing initiatives.

Reviewer: I don't know if we shouldn't use the four pillars here instead of this long laundry list (maybe pulling in from here: https://github.com/orcwg/orcwg/blob/3175352b45c87cc107825f5d19b15b20e27f0c66/program-plan.md, but without the metrics). @jrico-eclipse WDYT?

> ## How to Get Involved

Reviewer: I'd add our key resources there:

The SIG, the CRA hub and maybe the FAQ?

Author: see: https://github.com/orcwg/.github/pull/1/commits/fd5876f97187bb12ba9f1bcd3a2b55c30fb111f4

**[Join us and contribute to the Open Regulatory Compliance Working Group by:](https://orcwg.org/participate/)**
*  **Engaging in discussions** via our [Mailing List](https://accounts.eclipse.org/mailing-list/open-regulatory-compliance) or our [Chat Space](https://join.slack.com/t/orcwg/shared_invite/zt-2vi7gi5ad-re2b35i95ar3WaVF2zoZaA)
*  **Becoming a member** of the Working Group - [Contat Us](https://orcwg.org/membership/#contact-us-about-membership) for more details and refer to the [Working Group Charter](https://www.eclipse.org/org/workinggroups/open-regulatory-compliance-charter.php) 
*  **Exploring our repositories**:
   *  The Main GitHub Repository: [orcwg](https://github.com/orcwg/orcwg)
   *  CRA-Specific GitHub Repository: [cra-hub](https://github.com/orcwg/cra-hub)
* **Staying informed** through:
  *  [Mailing-list Archive](https://www.eclipse.org/lists/open-regulatory-compliance/maillist.html)
  *  [Community Calender](https://calendar.google.com/calendar/u/0/embed?src=c_7db8e3f13c4fac984103918a97c704bb1d619da0fdb66d33f1747849b6020aea@group.calendar.google.com)
  *  [Working Group Calls](https://github.com/orcwg/orcwg/blob/main/MEETINGS.md)

>   *  [Webinar Series](https://youtube.com/playlist?list=PLy7t4z5SYNaQEyXv253SI6PDdPjbYXcNV&si=FNi1Ib--TqpaemIg)

Reviewer: Could we use the content from https://github.com/orcwg/orcwg?tab=readme-ov-file#get-involved for consistency and maintainability?

> * **Going beyond** with the **Eclipse Community Newsletter**: [Subscribe Here](https://eclipse.us6.list-manage.com/subscribe?u=eaf9e1f06f194eadc66788a85&id=46e57eacf1&v_id=53&f_id=00fac2e1f0)

Reviewer: Let's focus on ORC, here and not on broader Eclipse marketing.
```suggestion
```

  </picture>
</a>

<!-- Made with [OSS Insight](https://ossinsight.io/) -->

<!-- Copy-paste in your Readme.md file -->

<a href="https://next.ossinsight.io/widgets/official/compose-org-overview-stats?activity=issues&owner_id=189788385&period=past_12_months" target="_blank" style="display: block" align="center">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://next.ossinsight.io/widgets/official/compose-org-overview-stats/thumbnail.png?activity=issues&owner_id=189788385&period=past_12_months&image_size=2x2&color_scheme=dark" width="177" height="auto">
    <img alt="[Overview] issues of eclipse-tractusx" src="https://next.ossinsight.io/widgets/official/compose-org-overview-stats/thumbnail.png?activity=issues&owner_id=189788385&period=past_12_months&image_size=2x2&color_scheme=light" width="177" height="auto">
  </picture>
</a>

<!-- Made with [OSS Insight](https://ossinsight.io/) -->

> 
> ## License
> 
> - [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) 

Reviewer: Already in the repo and slightly confusing (is it just for this content of for the whole content of the org?) I'd remove it.
```suggestion

```
